front/lib/api/assistant/conversation.ts

```diff
@@ -693,16 +701,29 @@ export async function* postUserMessage(
   }
 
   const results = await Promise.all([
-    removeNulls(
-      await Promise.all(
-        mentions.filter(isAgentMention).map((mention) => {
-          return getLightAgentConfiguration(auth, mention.configurationId);
-        })
-      )
+    await Promise.all(
+      mentions.filter(isAgentMention).map((mention) => {
+        return getLightAgentConfiguration(auth, mention.configurationId);
+      })
     ),
     await createOrUpdateParticipation(),
   ]);
-  const agentConfigurations = results[0];
+
```

Comment:
Note: in `edit/postUserMessage`, agents to which the user is not authorized are already filtered out. 

Still, we check and fail if one of the agents was not accessible, since it seems better to be explicit about it to the user than silently ignore the mention. 

The other option would be to use the new `canAccessConversation` fn relying on conversation participants. It would be somewhat cleaner, but makes the calls to `postUserMessage` more heavy, and since it's one of the most used and critical part of our app the choice I made here is to keep it lighter; happy to discuss